docker_cli/utils.py

```diff
@@ -0,0 +1,22 @@
+import random
+from string import ascii_lowercase, digits
+
+
+def generate_codename(k: int = 10) -> str:
```

Comment:
Not sure I'm a fan of this function name. I'd prefer something more descriptive.

```suggestion
def generate_random_string(k: int = 10) -> str:
```